test/controllers/update/secure.update.filter.controller.spec.ts

```diff
@@ -128,4 +147,40 @@ describe("SECURE UPDATE FILTER controller", () => {
       expect(resp.text).toContain(SERVICE_UNAVAILABLE);
     });
   });
+
+  describe("POST tests for remove journey", () => {
+    test(`redirect to ${UPDATE_INTERRUPT_CARD_URL}${JOURNEY_REMOVE_QUERY_PARAM} when no is selected`, async () => {
+      mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE_FLAG_ENABLE_UPDATE_TRUSTS
+
+      const resp = await request(app)
+        .post(`${SECURE_UPDATE_FILTER_URL}${JOURNEY_REMOVE_QUERY_PARAM}`)
+        .send({ is_secure_register: '0' });
+
+      expect(resp.status).toEqual(302);
+      expect(resp.header.location).toEqual(`${UPDATE_INTERRUPT_CARD_URL}${JOURNEY_REMOVE_QUERY_PARAM}`);
+      expect(mockSetExtraData).toHaveBeenCalledTimes(1);
+    });
+
+    test(`redirect to ${UPDATE_USE_PAPER_URL}${JOURNEY_REMOVE_QUERY_PARAM} when yes is selected`, async () => {
+      mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE_FLAG_ENABLE_UPDATE_TRUSTS
+
+      const resp = await request(app)
+        .post(`${SECURE_UPDATE_FILTER_URL}${JOURNEY_REMOVE_QUERY_PARAM}`)
+        .send({ is_secure_register: '0' });
```

Comment:
both the tests seem to send '0' which equates to No, but one of the tests says it is testing if Yes is selected, so it should send '1' for that one

Reply:
Fixed